include/ocpp/v201/database_handler.hpp

```diff
@@ -121,6 +121,12 @@ class DatabaseHandler : public common::DatabaseHandlerCommon {
 
     /// \brief Inserts or updates the given \p profile to CHARGING_PROFILES table
     virtual void insert_or_update_charging_profile(const int evse_id, const v201::ChargingProfile& profile);
+
+    /// \brief Deletes the profile with the given \p profile_id
+    virtual void delete_charging_profile(const int profile_id);
```

Comment:
I'm not sure what benefit we get from making profile_id const. It is already pass-by-value and so the original value won't change. I don't recall if Everest has a code standard that asks for it.

Reply:
I do believe this was a copy+paste. I did not change it for the exact "code standard" being unknown.